packages/data/src/onboarding/selectors.ts

```diff
@@ -21,6 +21,10 @@ export const getTasksStatus = (
 	return state.tasksStatus || {};
 };
 
+export const getTaskLists = ( state: OnboardingState ): TaskList[] => {
+	return state.taskLists || [];
```

Comment:
We shouldn't return a newly created empty array within the selector as it breaks memoization while `taskLists` is not set.
You could define an empty array outside of the function and return that, or use the `createSelector` memoization function.